src/dev-app/datepicker/datepicker-demo.html

```diff
@@ -138,8 +138,14 @@ <h2>Datepicker with value property binding</h2>
   <mat-datepicker-toggle [for]="datePicker4"></mat-datepicker-toggle>
   <mat-form-field>
     <mat-label>Value binding</mat-label>
-    <input matInput [matDatepicker]="datePicker4" [value]="date" [min]="minDate"
-           [max]="maxDate" [matDatepickerFilter]="filterOdd ? dateFilter : null">
+      <!--
+        TODO for reviewer: Angular sets the inputs for both directives on the
```

Comment:
We'll need to talk about this at the framework meeting

Reply:
Was there an outcome from the discussion of this at the framework meeting? This is an issue that I'm still running into and I don't have a solution.